src/TableTransforms.jl

```diff
@@ -39,6 +39,7 @@ export
   Coalesce,
   Identity,
   Center,
+  Replace,
```

Comment:
Move it to before Identity. Everything below Identity is more "numerical".

Reply:
OK

Reply:
Made in [0159ff2](https://github.com/JuliaML/TableTransforms.jl/pull/42/commits/0159ff21165a22578f40d5d28e94e37a44c31b66)